pennylane/devices/qubit/measure.py

```diff
@@ -194,7 +194,7 @@ def get_measurement_function(
                 return full_dot_products
 
             backprop_mode = math.get_interface(state, *measurementprocess.obs.data) != "numpy"
-            if isinstance(measurementprocess.obs, (LinearCombination, Hamiltonian)):
+            if isinstance(measurementprocess.obs, (Hamiltonian, LinearCombination)):
```

Comment:
Things like this is typically due to me changing something and then changing it back 🤣 but didn't pay attention to what the original order was.